.shippable.yml

```diff
@@ -37,7 +37,7 @@ build:
     - _make CFG_WITH_PAGER=y
     - _make CFG_WITH_PAGER=y CFG_CRYPTOLIB_NAME=mbedtls CFG_CRYPTOLIB_DIR=lib/libmbedtls
     - _make CFG_WITH_PAGER=y CFG_WITH_LPAE=y
-    - _make CFG_WITH_LPAE=y
+    - _make CFG_WITH_LPAE=y CFG_CORE_ASLR=y
```

Comment:
Would be nice to have a build in 32bit MMU with ASLR, i.e here `_make CFG_CORE_ASLR=y`.